md_api/src/main/java/com/wipro/ats/bdre/md/api/TermJob.java

```diff
@@ -86,7 +84,8 @@ public TermJobInfo execute(String[] params) {
             //The TermJob completes even if sending message fails
             try {
                 BasicConfigurator.configure();
-                new StatusNotification(termMessage, MDConfig.getProperty("status-notification.term-queue"));
+                StatusNotification statusNotification = new StatusNotification(termMessage, MDConfig.getProperty("status-notification.term-queue"));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this instantiation of "StatusNotification". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2440)